torch/distributed/fsdp/fully_sharded_data_parallel.py

```diff
@@ -126,7 +126,8 @@ class ShardingStrategy(Enum):
                    before forward computation and keeps the full parameters in
                    GPU memory until backward computation is done. It inserts reduce_scater
                    after backward computation for synchronizing and sharding gradients.
-                   Sharded optimizer states are updated locally.
+                   Sharded optimizer states are updated locally. Inside ``no_sync()``,
```

Comment:
I wonder if the beginning of this doc should be updated too, as parameters are still technically sharded, they are just in their full states after forward and through the backward

Reply:
Good point. I will update this doc (soon).